hugegraph-core/src/main/java/com/baidu/hugegraph/task/HugeTask.java

```
@@ -642,28 +636,14 @@ private void checkPropertySize(int propertyLength, String propertyName) {
     }
 
     public void syncWait() {
-        HugeTask task = null;
+        HugeTask task;
         try {
-            Id id = this.id();
             do {
-                task = this.scheduler().task(id);
-                if (task.status.code() > TaskStatus.QUEUED.code()) {
-                    task.get();
-                }
+                task = this.scheduler().task(this.id());
             } while (!task.completed());
             assert task.completed();
-        } catch (ExecutionException e) {
-            Throwable cause = e.getCause();
-            if (cause instanceof RuntimeException) {
-                throw (RuntimeException) cause;
-            }
-            throw new HugeException("Async task failed with error: %s",
-                                    cause, cause.getMessage());
         } catch (Exception e) {
-            throw new HugeException("Async task failed with error: %s",
-                                    e, e.getMessage());
-        } finally {
-            ((StandardTaskScheduler) task.scheduler()).remove(id);
+            // ignore
```

don't ignore exception here